src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -643,6 +643,21 @@
       <summary>Creates an BaseExpressionSyntax node.</summary>
     </FactoryComment>
   </Node>
+  <Node Name="OriginalExpressionSyntax" Base="InstanceExpressionSyntax">
```

Comment:
@gafter I'll keep that approach in mind. For now, this follows the approach used for `AwaitExpressionSyntax`.